src/ansys/mapdl/core/cli/hpc.py

```diff
@@ -0,0 +1,271 @@
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""Submit PyHPS jobs to a cluster"""
+
+import logging
+import os
+from typing import Optional, Union
+
+import click
+
+from ansys.mapdl.core.cli import main
+
+logger = logging.getLogger()
+
+
+@main.command(
+    short_help="Submit jobs to an HPC cluster using PyHPS package.",
+    help="""Submit jobs to an HPC cluster using PyHPS package.""",
+)
+@click.argument("main_file")
+@click.option(
+    "--name",
+    default=None,
+    type=str,
+    help="""Name of the PyHPS project to be created.""",
+)
+@click.option(
+    "--url",
+    default=None,
+    type=str,
+    help="""URL where the HPS cluster is deployed. For example: "https://myserver:3000/hps" """,
+)
+@click.option(
+    "--user", default=None, type=str, help="Username to login into the HPC cluster."
+)
+@click.option(
+    "--password",
+    default=None,
+    type=str,
+    help="Password used to login into the HPC cluster.",
+)
+@click.option(
+    "--python",
+    default=None,
+    type=str,
+    help="""Set python version to be used to create the virtual environment and
+run the python file. By default it uses python3 (default in cluster).""",
+)
+@click.option(
+    "--output_files",
+    default=None,
+    type=str,
+    help="""Set the output files to be monitored.""",
+)
+@click.option(
+    "--shell_file",
+    default=None,
+    type=str,
+    help="""If desired, you can provide a shell script to execute instead of
+the python file. You can call your python file from it if you wish. By default,
+it is not used.""",
+)
+@click.option(
+    "--requirements_file",
+    default=None,
+    type=str,
+    help="""If provided, the created virtual environment is installed with the
+libraries specified in this file. If not, the activated virtual environment is
+cloned through a temporary 'pip list' file. If you are using editable package,
+it is recommended you attach your own requirement file using ``pip freeze`` """,
```

Comment:
```suggestion
    help="""Optional created virtual environment to install with the
libraries specified in this file. If not, the activated virtual environment is
cloned through a temporary ``pip list`` file. If you are using an editable package,
you should attach your own requirement file using ``pip freeze`` """,
```
What file is meant by "specified in this file"? It is unclear.